normalize_repos/normalize_repos.py

```diff
@@ -26,10 +29,16 @@ def get_cc_repos(github):
     cc = get_cc_organization(github)
     return cc.get_repos()
 
+def check_project_status(repo):
+    contents = repo.get_contents(".cc-metadata.yml")
+    contents = contents.decoded_content
+    metadata = yaml.load(contents, Loader=yaml.FullLoader)
+    return metadata["engineering_project"]
 
 def update_branch_protection(repo):
     master = repo.get_branch("master")
-    if repo.name not in branch_protections.EXEMPT_REPOSITORIES:
+    status = check_project_status(repo)
+    if repo.name not in branch_protections.EXEMPT_REPOSITORIES or status == False:
```

Comment:
Avoid equality checks on Booleans. Use them directly or invert them with `not`.
```suggestion
    if repo.name not in branch_protections.EXEMPT_REPOSITORIES or not is_engineering_project(repo):
```

Reply:
Got it. Done.